ios/RCTMGL/MGLModule.m

```diff
@@ -0,0 +1,78 @@
+//
+//  MGLModule.m
+//  RCTMGL
+//
+//  Created by Nick Italiano on 8/23/17.
+//  Copyright © 2017 Mapbox Inc. All rights reserved.
+//
+
+#import "MGLModule.h"
+#import "RCTMGLEventTypes.h"
+@import Mapbox;
+
+@implementation MGLModule
+
+RCT_EXPORT_MODULE();
+
+- (NSDictionary<NSString *, id> *)constantsToExport
+{
+    // style urls
+    NSMutableDictionary *styleURLS = [[NSMutableDictionary alloc] init];
+    [styleURLS setObject:[MGLStyle.streetsStyleURL absoluteString] forKey:@"Street"];
+    [styleURLS setObject:[MGLStyle.darkStyleURL absoluteString] forKey:@"Dark"];
+    [styleURLS setObject:[MGLStyle.lightStyleURL absoluteString] forKey:@"Light"];
+    [styleURLS setObject:[MGLStyle.outdoorsStyleURL absoluteString] forKey:@"Outdoors"];
+    [styleURLS setObject:[MGLStyle.satelliteStyleURL absoluteString] forKey:@"Satellite"];
```

Comment:
There are a few more styles missing from this list: https://www.mapbox.com/api-documentation/#styles

Reply:
Updated